Content.Server/Nyanotrasen/Mail/MailSystem.cs

```diff
@@ -104,6 +109,20 @@ public override void Update(float frameTime)
             }
         }
 
+        /// <summary>
+        /// Dynamically add the MailReceiver component to appropriate entities.
+        /// </summary>
+        private void OnSpawnPlayer(PlayerSpawningEvent args)
+        {
+            if (args.SpawnResult == null ||
+                args.Job == null )
+            {
+                return;
+            }
+
+            AddComp<MailReceiverComponent>(args.SpawnResult.Value);
```

Comment:
this is going to explode if something already has the mail reciever component. Need to do EnsureComp instead.

Reply:
Its the same as delta, i removed all reciever component from yml, this isnt an issue anymore.

Reply:
ok well it could be an issue in the future, it needs to be EnsureComp, not AddComp